src/main/java/com/noroff/mefit/model/Profile.java

```diff
@@ -0,0 +1,83 @@
+package com.noroff.mefit.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.Size;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "profile", schema = "public")
+public class Profile {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Getter
+    @Setter
+    @Column(name = "id")
+    private Integer id;
+
+    @Getter @Setter
+    @Size(min = 1, max = 50)
```

Comment:
What is our userId limit? How large is our database?  
userID: 99999999999999999999999999999999999999999999999999 seems unlikely..
```suggestion
```